crates/configuration/src/relaychain.rs

```diff
@@ -44,42 +44,52 @@ pub struct RelaychainConfig {
 }
 
 impl RelaychainConfig {
+    /// The chain name.
     pub fn chain(&self) -> &Chain {
         &self.chain
     }
 
+    /// The default command used for nodes.
     pub fn default_command(&self) -> Option<&Command> {
         self.default_command.as_ref()
     }
 
+    /// The default container image used for nodes.
     pub fn default_image(&self) -> Option<&Image> {
         self.default_image.as_ref()
     }
 
+    /// The default resources limits used for nodes.
     pub fn default_resources(&self) -> Option<&Resources> {
         self.default_resources.as_ref()
     }
 
+    /// The default database snapshot location that will be used for state.
     pub fn default_db_snapshot(&self) -> Option<&AssetLocation> {
         self.default_db_snapshot.as_ref()
     }
 
+    /// The default args that will be used to launch the node command.
     pub fn default_args(&self) -> Vec<&Arg> {
         self.default_args.iter().collect::<Vec<&Arg>>()
     }
 
+    /// The location of an pre-existing chain spec for the relaychain.
     pub fn chain_spec_path(&self) -> Option<&AssetLocation> {
         self.chain_spec_path.as_ref()
     }
 
-    pub fn random_minators_count(&self) -> Option<u32> {
+    // TODO: description?
```

Comment:
Something in the lines of...
```suggestion
    /// The number of `random nominators` to create for chains using staking, this is used in tandem with `max_nominators` to simulate the amount of nominators and nominations.
```

Reply:
I think we should discuss about it given the other one can be None, should they both be set to some value together or this one can be None and the other set but not the other way ?

Reply:
Yes, we can touch base about these fields since are very specific to an advance use-case and I think user that set those are aware of how `npos` works. Thx!

Reply:
I have update to: ```The number of `random nominators` to create for chains using staking, this is used in tandem with `max_nominations` to simulate the amount of nominators and nominations.```